composer.json

```diff
@@ -20,6 +20,8 @@
     ],
     "require": {
         "php":                                  ">=5.5.9",
+        "ext-exif":                             "*",
```

Comment:
Exif extension is not required by any package, but if it is missing, it causes error during `cache:warmup`

```
php app\console cache:warmup --env=test

Warming up the cache for the test environment with debug true
                                                                
  [Imagine\Exception\NotSupportedException]                     
  PHP exif extension is required to use the ExifMetadataReader  
                                                                
 
 ```